Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            const int numOfPerson = 2;
+
+            for (var num = 0; num < numOfPerson; num++)
+            {
+                Console.WriteLine("Calculate BMI for a person.");
+
+                var name = PromptString("Enter name: ");
+                var surname = PromptString("Enter surname: ");
+                var age = PromptInt("Enter age: ");
+                var weightKg = PromptFloat("Enter weight (kg): ");
+                var heightCm = PromptFloat("Enter height (cm): ");
+
+                Console.WriteLine(name + " " + surname + " is " + age + " years old, his weight is " + weightKg + " kg and his height is " + heightCm + " cm.");
+
+                float bmi = CalculateBmi(weightKg, heightCm / 100);
+                if (bmi != -1) Console.WriteLine("Body-mass index (BMI) is " + bmi + ".");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4885449)